eng/nuget-client-versions.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
I think it would be nice if the three json files had a similar naming convention.  I understand nuget-config.json is confusing.  `lcu-info.json`, `nuget-client.json`, `vs-info.json`